src/app/components/media/MediaComponent.js

```diff
@@ -67,18 +112,31 @@ class MediaComponent extends Component {
     }
   }
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps, prevState) {
     this.setCurrentContext();
     MediaComponent.scrollToAnnotation();
     if (this.props.media.dbid !== prevProps.media.dbid) {
       this.subscribe();
     }
+    if (prevState.playerRef !== this.state.playerRef) this.setPlayerRect();
   }
 
   componentWillUnmount() {
+    window.removeEventListener('resize', this.onWindowResize);
     this.unsubscribe();
   }
 
+  onWindowResize = () => {
+    this.setPlayerRect();
+  }
+
+  onTimelineCommentOpen = (fragment) => {
+    // this call will come from Annotation.js
+    if (!fragment) return;
+    const parsedFragment = parseInt(fragment.substring(2), 10);
+    this.setState({ seekTo: parsedFragment, showVideoAnno: 'timeline' });
```

Comment:
This looks like it meant `showVideoAnno: true, videoAnnotationTab: 'timeline'`, right?

Reply:
Fixed.